client/src/components/deals.jsx

```diff
@@ -1,38 +1,19 @@
-const React = require('react');
-// const styled = require('styled-components');
-// const Deal = require('./deal.jsx');
+import React from 'react';
+import Deal from './deal';
 
 const Deals = (props) => (
-  // const labelWrapper =styled.label `
-  //   border
-  // `;
-  props.deals.map((deal) => (
-    <div key={deal.id}>
-      <label>
-        <input
-          id={deal.id}
-          type="radio"
-          name="packageDeals"
-          value={deal.title}
+  props.deals.map((deal, i) => {
```

Comment:
Yeah, great work on pulling the map function back into the `deals.jsx` file @brkidder !